src/components/task-plan/homework/exercises.cjsx

```diff
@@ -190,24 +269,46 @@ AddExercises = React.createClass
       i += 2
     rows
 
+  renderSection: (key) ->
+    section = TocStore.getSectionLabel(key)
+    if not section
+      return <span></span>
+
+    <BS.Row>
+      <BS.Col xs={12}>
+        <label className='-exercises-section-label'>
+          {section.chapter_section[0]} - {section.chapter_section[1]}.
```

Comment:
Can this code use the old `chapter_section` format? and then we can make a global change in a separate PR? Affects learning guide and mock JSON files.